### DISCUSSION TITLE
How do I pick the opposite theme (assuming some logic)

### DISCUSSION THREAD

**Discussion by herzaso** [tailwindlabs/tailwindcss#13337] [General]
I have two themes - `light` and `dark`, but some components should be in the opposite theme (for example, a tooltip in light theme usually has a dark background and in dark theme should have a light background).

Right now,, I use the class theme and assuming light and dark themes, I can add the `dark` class name to indicate that I want the component to be in dark theme but instead, I want to use `opposite-theme` class (name TBD) to indicate that in light theme, I want it to be `dark` and in dark theme I want it to be `light`.

Is there a way to achieve that?

**Comment by andymccray** [comment]
Assuming that instance of the tooltip (for example) has inverse styling in all contexts can you just use the `light` and `dark` modifiers to _visually_ invert it? For example you’d apply `dark:bg-white` because _this is how tooltips look in dark mode_.
